Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -953,10 +954,10 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
         if (!download)
             return;
 
-        NSData *resumeData = nil;
-        if (id userInfo = error.userInfo) {
+        RetainPtr<NSData> resumeData;
+        if (RetainPtr<id> userInfo = updatedError.get().userInfo) {
             if ([userInfo isKindOfClass:[NSDictionary class]]) {
```

Comment:
Can we do something more like this?
```suggestion
        if (RetainPtr userInfo = dynamicObjCCast<NSDictionary>(updatedError.get().userInfo)) {
```

Reply:
It's `dynamic_objc_cast` but I think so, yes. Will update.